x-pack/plugins/security_solution/public/management/pages/policy/types.ts

```diff
@@ -61,7 +61,7 @@ export interface PolicyDetailsState {
   /** current location of the application */
   location?: Immutable<AppLocation>;
   /** A summary of stats for the agents associated with a given Fleet Agent Policy */
-  agentStatusSummary?: GetAgentStatusResponse['results'];
+  agentStatusSummary?: Omit<GetAgentStatusResponse['results'], 'updating'>;
```

Comment:
Curious why only this reference had to be updated. Looks like we use `GetAgentStatusResponse['results']` in a few other places through our code.

From a Policy Details page standpoint, this seems ok.

@kevinlog looks like this PR changes the output of agent status to introduce `updating` (whose counts used to be going into the `other` grouping). Does that impact our Onboarding prompts or the "endpoint are being installed" message on the endpoints list??

Reply:
@paul-tavares I think it depends if `updating` is still considered active.  We should manually test this against our code which would be the quickest way to go about it.  It should be as simple to as checking out this code and standing up an Endpoint.  In addition, we should update a policy to see how that affects things.

Reply:
Agents will not be in `updating` if the policy is updated only when the agent `enroll` (before getting the first policy) upgrade or `unenroll`(before confirming the unenrollment)

Reply:
@kevinlog @paul-tavares let me know if you did a manual test here to check that is not breaking endpoint features, before I merge

Reply:
Tested the two use cases in Endpoint with a real `elastic-agent` / `elastic-endpoint` on a Win VM and it all seems good as far as this change goes.